zipline/data/us_equity_loader.py

```diff
@@ -121,7 +136,7 @@ def _get_adjustments_in_range(self, assets, days, field):
                 for m in mergers:
                     dt = m[0]
                     if start < dt <= end:
-                        end_loc = days.get_loc(dt)
+                        end_loc = dts.searchsorted(dt)
```

Comment:
out of scope for this PR, but would love to see less duplicates of the intricate logic around each merger, dividend, and split (all the `Float64Multiply` construction, etc).  I'll submit a PR afterwards.